### PR TITLE
CSS style: Don't hide focus indicators

### PR DESCRIPTION
**Pull request by Jookia**
Hiding focus indicators is unhelpful for people navigating by keyboard.
Show them instead.

### Before submitting a pull-request to GitBucket I have first:

- [x] read the [contribution guidelines](https://github.com/gitbucket/gitbucket/blob/master/.github/CONTRIBUTING.md)
- [x] rebased my branch over master
- [x] verified that project is compiling
- [x] verified that tests are passing
- [x] squashed my commits as appropriate *(keep several commits if it is relevant to understand the PR)*
- [x] [marked as closed using commit message](https://help.github.com/articles/closing-issues-via-commit-messages/) all issue ID that this PR should correct

### CODE REVIEW

File: src/main/webapp/assets/vendors/AdminLTE-2.4.2/css/AdminLTE.css [hunk 1]
```diff
@@ -172,13 +172,6 @@ h6,
 a {
   color: #337ab7;
 }
-a:hover,
-a:active,
-a:focus {
-  outline: none;
-  text-decoration: none;
-  color: #23527c;
-}
```
Comment (SIkebe) [inline src/main/webapp/assets/vendors/AdminLTE-2.4.2/css/AdminLTE.css:181]: What for?

BTW, do not modify this css file directly as it is from third party library.  Anything you modified would be lost when AdminLTE upgraded.

Comment (SIkebe) [reply src/main/webapp/assets/vendors/AdminLTE-2.4.2/css/AdminLTE.css:181]: > What for?
> 

Sorry, you clarified here
>Hiding focus indicators is unhelpful for people navigating by keyboard.
Show them instead.

Comment (SIkebe) [reply src/main/webapp/assets/vendors/AdminLTE-2.4.2/css/AdminLTE.css:181]: Hmm, why  did you remove `color: #23527c;`? Screenshots before and after would help.

**Comment by Jookia** [comment]
Should font color change during hovering or focus? I'll add it back and
check it later today.

File: src/main/webapp/assets/vendors/AdminLTE-2.4.2/css/AdminLTE.css (hunk 1 above)
Comment (Jookia) [reply src/main/webapp/assets/vendors/AdminLTE-2.4.2/css/AdminLTE.css:181]: How else do I modify this CSS? It seems that the way AdminLTE indicates focus is by slightly darkening links. I can't even see the difference. So this commit replaces it with classic browser focus stuff. You can compare it by tabbing through the GitBucket demo and https://git.xogium.me/

**Comment by SIkebe** [comment]
Ah I see, this color change is not from AdminLTE but Bootstrap. So your deletion does not have any effect.

color
-----

This `#23527c` is applied.

image

**Comment by Jookia** [comment]
Ah, that's good. My intention is purely to make it so the focus box shows around things when using keyboard navigation, so you can see where you're at in the page.

**Comment by SIkebe** [comment]
If your link has underline in it, this PR makes it harder to read correctly. Could you also tweak padding?

underline

**Comment by Jookia** [comment]
Sure, how?

**Comment by Jookia** [comment]
Ok I managed to reproduce it on my machine. I can't change that without changing the font used in Gitbucket. Maybe I should add the underline back?

**Comment by SIkebe** [comment]
>Edit: For what it's worth I can't get your padding behavior on my browser.

That's strange... which browser are you using? As I said before screenshots would help.

**Comment by SIkebe** [comment]
>I can't change that without changing the font used in Gitbucket.

How did you verify that? You can override any third party css behavior if you can define your style in gitbucket.css.
**Note: Do not use below style definition as it is. It affects all `<a>` tag.**

image

**Comment by Jookia** [comment]
On Sat, Jul 04, 2020 at 10:12:02PM -0700, SIkebe wrote:
> >I can't change that without changing the font used in Gitbucket.
> 
> How did you verify that? You can override any third party css behavior if you can define your style in gitbucket.css.
> **Note: Do not use below style definition as it is. It affects all `<a>` tag.**
> 
> image

I disabled the link's font CSS in the Firefox inspector and it showed a
better font with better spacing for the underscores.

**Pull request closed without merging**